src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/List.js

```diff
@@ -65,6 +67,7 @@ const USER_SETTING_ADAPTER = 'adapter';
 @observer
 class List extends React.Component<Props> {
     static defaultProps = {
+        actions: [],
```

Comment:
think this should be added to the `README.md`

Reply:
yes you are right, but it looks like the `README.md` does not contain most other props like `adapterOptions`/`paginated`/`selectable`/... too 😕